docs/data/charts/pie-demo/OnSeriesItemClick.js

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import { PieChart } from '@mui/x-charts/PieChart';
 
```

Comment:
Can't remove this newline for some reason. But didn't take the time to look into it.

Reply:
This behavior is known and we tend to ignore it. 🙈 
P.S. I take it that adding `type` after `import` doesn't change anything? 🤔

Reply:
It signals that this import is not usable as a javascript value. Can be important depending on the context this is compiled in.